src/DistributionsBase.jl

```diff
@@ -44,3 +44,13 @@ from manifold of type `TM`.
 """
 abstract type TVectorDistribution{TM<:Manifold, T} <: Distribution{TVectorvariate, TVectorSupport{TM, T}}
 end
+
+
+"""
+    get_support(d::TVectorDistribution)
+
+Get the object of type `TVectorSupport` for the distribution `d`.
+"""
```

Comment:
Could. we maybe rename that to `domain`? `get_support`sounds a little bit like it starts a phone_call.

Reply:
OK, done